controllers/market/poster.js

```diff
@@ -0,0 +1,359 @@
+const schedule = require('node-schedule');
+const { Market, Warehouse, Bid, PlayerFantasy, Op } = require('../../databases/db');
+const responses = require('../../utils/responses/responses');
+
+function padTo2Digits(num) {
+    return num.toString().padStart(2, '0');
+  }
+
+function formatDate(date) {
+    return (
+      [
+        date.getFullYear(),
+        padTo2Digits(date.getMonth() + 1),
+        padTo2Digits(date.getDate()),
+      ].join('-') +
+      ' ' +
+      [
+        padTo2Digits(date.getHours()),
+        padTo2Digits(date.getMinutes()),
+        padTo2Digits(date.getSeconds()),
+      ].join(':')
+    );
+  }
+
+const poster = async (req, res) => {
+    const userId = req.user.id.id;
+    const eventId = req.eventId;
+    const timeNow = new Date(Date.now());
+    const endTime = new Date(timeNow.getTime() + 180000);
+    let { initialValue, directPurchase } = req.body;
+    const {playerId} = req.body
+    const warehouse = await Warehouse.findOne({
+        raw: true,
+        where: {
+            [Op.and]: [{
+                stickerId: playerId
+            }, {
+                eventId: eventId
+            },{ 
+                userId: userId
+            }]
+        }
+    });
+    const quant = warehouse.quantity;
+
+    if (!warehouse) {
+        return responses.errorDTOResponse(res, 404, 'No posees este Sticker')
+    }
+    const isInLineup = warehouse.isInLineup;
+    if (isInLineup) {
```

Comment:
Aqui hay un problema, que pasa si yo tengo 2 messis, uno esta en mi alineacion y el otro lo quiero subastar, entonces no me va a dejar subastar el que no lo tengo en mi alineacion por que tengo uno que si xD

@DaikoneKisu las desventajas de fusionar las tablas v:

Igual... CREO se puede arreglar:
```suggestion
    if (isInLineup || warehouse.quantity >= 2) {
```

Reply:
0.0